insights/parsers/kpatch_list.py

```diff
@@ -0,0 +1,67 @@
+"""
+KpatchList - command ``/usr/sbin/kpatch list``
+==============================================
+
+The ``/usr/sbin/kpatch list`` command provides information about
+the installed patch modules.
+
+Sample content from command ``/usr/sbin/kpatch list`` is::
+    Loaded patch modules:
+    kpatch_3_10_0_1062_1_1_1_4 [enabled]
+
+    Installed patch modules:
+    kpatch_3_10_0_1062_1_1_1_4 (3.10.0-1062.1.1.el7.x86_64)
+
+Examples:
+    >>> 'kpatch_3_10_0_1062_1_1_1_4' in kpatchs.loaded
+    True
+    >>> kpatchs.loaded.get('kpatch_3_10_0_1062_1_1_1_4')
+    'enabled'
+    >>> 'kpatch_3_10_0_1062_1_1_1_4' in kpatchs.installed
+    True
+    >>> kpatchs.installed.get('kpatch_3_10_0_1062_1_1_1_4')
+    '3.10.0-1062.1.1.el7.x86_64'
+"""
+
+from insights.specs import Specs
+from insights.parsers import SkipException, ParseException
+from insights import parser, CommandParser
+
+
+@parser(Specs.kpatch_list)
+class KpatchList(CommandParser):
+    """Class for command: /usr/sbin/kpatch list"""
+    def parse_content(self, content):
+        if not content:
+            raise SkipException("No Data from command: /usr/sbin/kpatch list")
+
+        self._loaded = {}
+        self._installed = {}
+        cur_dict = {}
+        for line in content:
+            if not line.strip():
+                continue
+
+            if 'Loaded patch modules' in line:
+                cur_dict = self._loaded
+                continue
+
+            if 'Installed patch modules' in line:
+                cur_dict = self._installed
+                continue
+
+            kpatch, info = [k.strip('()[]') for k in line.split()]
+            if not info:
+                raise ParseException("Parser Error: Invalid Line in content of 'kpatch list'")
+
+            cur_dict[kpatch] = info
+
+    @property
+    def loaded(self):
+        """``dict`` the loaded kpatchs"""
```

Comment:
The type of return value is not in this format.

Please check the examples in other parsers:

https://github.com/RedHatInsights/insights-core/blob/f58f26eb08bbea38fd4899e6b8a7eae056522c9f/insights/parsers/blkid.py#L76

or take a look at:

https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html